piqueserver/commands.py

```diff
@@ -297,11 +297,14 @@ def get_player(protocol, value, spectators=True):
 
 def get_team(connection, value):
     value = value.lower()
-    if value == 'blue':
+    team_1 = connection.protocol.team_1
+    team_2 = connection.protocol.team_2
+    team_spectator = connection.protocol.team_spectator
+    if value == team_1.name.lower():
         return connection.protocol.blue_team
-    elif value == 'green':
+    elif value == team_2.name.lower():
         return connection.protocol.green_team
-    elif value == 'spectator':
+    elif value == team_spectator.name.lower():
         return connection.protocol.spectator_team
     raise CommandError("Invalid Team")
```

Comment:
Shouldn't `ValueError()` be more appropriate?

Reply:
Oh, is it? I don't know, I didn't look at this part.